tests/unit/slack_annotations/format_test.py

```diff
@@ -143,3 +152,27 @@ def test_reply(self):
                 {"type": "plain_text", "text": "(None)"},
             ],
         }
+
+    def test_newline(self):
+        annotation = {
+            "user": "acct:test_user_1@hypothes.is",
+            "uri": "https://example.com/",
+            "links": {"incontext": "https://hyp.is/test_annotation_id_1/example.com/"},
+            "document": {"title": ["Annotation \n title"]},
```

Comment:
This is the issue we actually had on our own sign up page with newline in the title for some reason. But I can imagine other titles in the wild like this as well.